ui/page/account/account_page.go

```diff
@@ -0,0 +1,272 @@
+		layout.Rigid(func(gtx C) D {
+			txt := pg.Theme.Label(values.TextSize16, values.String(values.StrAddNewAccount))
+			txt.Color = pg.Theme.Color.DefaultThemeColors().Primary
+			txt.Font.Weight = font.SemiBold
+			return layout.Inset{
+				Left: values.MarginPadding8,
+			}.Layout(gtx, txt.Layout)
+		}),
+	)
+}
+
+func (pg *Page) accountItem(gtx C, account *sharedW.Account, isHiddenLine bool) D {
+	return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+		layout.Rigid(func(gtx C) D {
+			return layout.Inset{Top: values.MarginPadding36}.Layout(gtx, func(gtx C) D {
+				return layout.Flex{}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						txt := pg.Theme.Label(values.TextSize18, account.AccountName)
+						txt.Font.Weight = font.SemiBold
+						return txt.Layout(gtx)
+					}),
+					layout.Flexed(1, func(gtx C) D {
+						balance := account.Balance.Total.String()
+						totalBalance := pg.wallet.ToAmount(account.Balance.Total.ToInt()).ToCoin()
+						if pg.exchangeRate != -1 && pg.usdExchangeSet {
+							balanceUSD := utils.FormatAsUSDString(pg.Printer, utils.CryptoToUSD(pg.exchangeRate, totalBalance))
+							balance = fmt.Sprintf("%s (%s)", balance, balanceUSD)
+						}
+						txt := pg.Theme.Label(values.TextSize18, balance)
+						txt.Font.Weight = font.SemiBold
+						return layout.E.Layout(gtx, txt.Layout)
+					}),
+				)
+			})
+		}),
+		layout.Rigid(func(gtx C) D {
+			return layout.Inset{Top: values.MarginPadding16, Bottom: values.MarginPadding28}.Layout(gtx, func(gtx C) D {
+				return layout.Flex{Spacing: layout.SpaceBetween}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						txt := pg.Theme.Label(values.TextSize16, values.String(values.StrAmountSpendable))
+						txt.Font.Weight = font.SemiBold
+						txt.Color = pg.Theme.Color.GrayText3
+						return txt.Layout(gtx)
+					}),
+
+					layout.Flexed(1, func(gtx C) D {
+						spendable := account.Balance.Spendable.String()
+						spendableCoin := pg.wallet.ToAmount(account.Balance.Spendable.ToInt()).ToCoin()
+						if pg.exchangeRate != -1 && pg.usdExchangeSet {
+							balanceUSD := utils.FormatAsUSDString(pg.Printer, utils.CryptoToUSD(pg.exchangeRate, spendableCoin))
+							spendable = fmt.Sprintf("%s (%s)", spendable, balanceUSD)
+						}
+						txt := pg.Theme.Label(values.TextSize16, spendable)
+						txt.Color = pg.Theme.Color.GrayText3
+						txt.Font.Weight = font.SemiBold
+						return layout.E.Layout(gtx, txt.Layout)
+					}),
+				)
+			})
+		}),
+		layout.Rigid(func(gtx C) D {
+			if isHiddenLine {
+				return D{}
+			}
+			line := pg.Theme.Line(1, 0)
+			line.Color = pg.Theme.Color.Gray9
+			return line.Layout(gtx)
+		}),
+	)
+}
+
+// HandleUserInteractions is called just before Layout() to determine
+// if any user interaction recently occurred on the page and may be
+// used to update the page's UI components shortly before they are
+// displayed.
+// Part of the load.Page interface.
+func (pg *Page) HandleUserInteractions() {
```

Comment:
Since this is called before `Layout()`, it makes sense for this method to appear before the `Layout()` method.